<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1308</width>
    <height>984</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="sendStop">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendStart">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboPortNumber">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <item>
    <property name="text">
     <string>COM1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>COM7</string>
    </property>
   </item>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEditReport">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>1261</width>
     <height>851</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBaud">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>115200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>57600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>38400</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9600</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushClear">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>40</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>0.89486742, -0.003822158, -0.75105205, -1.3965429, -1.7378376, -2.1130203, -2.2132096, -2.2033471, -2.0241906, -1.5914581, -1.0499022, -0.57789108, -0.3131374, -0.20084404, -0.18379899, 0.016525057, 0.50921179, 0.77751147, 0.75067496, 0.78843003, 0.77137524, 0.73257374, 0.67985885, 0.62183591, 0.60469296, 0.58129958, 0.65998204, 0.63413413, 0.63558912, 0.76050608, 0.76912011, 0.73457027, 0.72108854, 0.75040232, 0.62612794, 0.51689273, 0.54554218, 0.50588773, 0.5258932, 0.49186076, 0.52836623, 0.62282844, 0.58039006, 0.54824181, 0.50770483, 0.51204975, 0.40906504, 0.31038144, 0.36845009, 0.38137005, 0.33419976, 0.18130539, 0.20183031, 0.2933171, 0.27945872, 0.29958438, 0.21474396, 0.18456487, 0.13197654, 0.057778811, 0.018044803, -0.071802363, 0.005535548, 0.066041, 0.042362944, 0.05525463, -0.005126899, -0.051986345, -0.015207146, -0.0094131254, 0.02094263, 0.092754939, 0.00073659051, -0.071986495, 0.070326647, 0.091355005, 0.028342719, 0.17460518, 0.28679751, 0.27782832, 0.31537371, 0.31602348, 0.36637824, 0.44613065, 0.41105086, 0.42461382, 0.4831252, 0.4544008, 0.46774525, 0.56968959, 0.54844061, 0.52878176, 0.63875397, 0.64354817, 0.62231549, 0.6227605, 0.57584803, 0.56640109, 0.56635399, 0.57809064, 0.61300844, 0.63278994, 0.59456305, 0.61561681, 0.72445263, 0.7123046, 0.67086495, 0.71278444, 0.66167141, 0.58293314, 0.58568763, 0.61013419, 0.69306756, 0.68844189, 0.61338458, 0.60209334, 0.42122252, 0.16604014, 0.14347481, 0.19632701, 0.16141046, 0.1275313, -0.018672703, -0.23865283, -0.3706969, -0.59290109, -0.63411579, -0.44082747, -0.5180703, -0.70243851, -0.89974628, -1.4415163, -1.8616119, -2.3213812, -2.8848769, -3.530246, -4.4125149, -3.9033308, -3.5671064, -1.5363381</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushSend">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>10</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Send One </string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendFileButton">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>40</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Send File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushEcho">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>241</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Echo </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushSave">
   <property name="geometry">
    <rect>
     <x>1144</x>
     <y>80</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Results</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
